package.json

```diff
@@ -12,17 +17,19 @@
     "git-pull": "*",
     "quarantine": "*",
     "redwrap": "*",
-    "request": "*",
+    "request": "2.54.0",
```

Comment:
Reason we depend on a particular version now, rather than the latest or at least the latest semi-major?